### PR TITLE
Basic layout

### PR DESCRIPTION
**Pull request by joshdabosh**
Here's what it looks on my side: image

The materials selector doesn't actually do anything to select the material, so it'll have to be rigged with javascript later.

### CODE REVIEW

**Comment by CLiu13** [comment]
(I will leave it up to @takore05 to merge once she reviews this)

**Comment by joshdabosh** [comment]
new font:
image

File: css/index.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	padding: 0;
+	margin: 4px;
+}
+
+.material-select:hover::before{
+	content:"> ";
+}
+
+.zen-button{
+	background-color: black;
```
Comment (takore05) [inline css/index.css:47]: the background color of the buttons are #3c7dfa

Comment (takore05) [reply css/index.css:47]: and the font color is #bae1ff

**Pull request merged**